src/dlnodesession.cpp

```diff
@@ -146,10 +146,12 @@ Status DLNodeSession::prepareInputsAndModelForInference() {
 Status DLNodeSession::validate(const InferenceEngine::Blob::Ptr& blob, const TensorInfo& info) {
```

Comment:
Did you consider going with `tensorInfo` instead of just `info` for param name?
Without looking at the type explicitly or a bit deeper code analysis it's hard to tell if this info is about pipeline, model, inputs, outputs, or some other info received as a result from another operation.